### DISCUSSION TITLE
4.18.0之后select组件的dropdownRender方法参数里的props属性变成了空对象 4.17.0是有值的 有哪里的值可以替代这个属性吗 我需要这个值对这个下拉框进行一些处理

### DISCUSSION THREAD

**Discussion by Monkey-Decade** [ant-design/ant-design#50304] [Q&A]
<img width="298" alt="image" src="https://github.com/user-attachments/assets/0d4ea8dd-b3c2-4ad7-8541-ae65152ca25b">

<img width="316" alt="image" src="https://github.com/user-attachments/assets/daaa703a-8db6-41d0-b2fc-a010bbf8d2ac">

<img width="315" alt="image" src="https://github.com/user-attachments/assets/9ed36ff6-1573-45df-86db-2710a4fbcf7d">

**Comment by Wxh16144** [comment]
> 有哪里的值可以替代这个属性吗

没有。

审查了一下代码，看到在 4.18.0 版本有对 rc-select 重构，并且文档只是说 dropdownRender 第一个参数是一个 originNode 是一个 React jsx 对象，并不建议再去读取属性 props 这些。所以移除了应该是合理的。

重构后的代码 originNode 还是一个组件，不过 state 都是用 context 获取了。（**很不建议的做法：** 导入 rc-select 的 context ，你也从这里面获取
